share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/spack.yaml

```diff
@@ -2,19 +2,14 @@ spack:
   view: false
 
   definitions:
-  - compiler_specs:
-    - gcc
-    - gettext
-
-  - compiler_target:
-    - '%gcc@7.3.1 target=aarch64'
-
   - optimized_configs:
-    - gromacs
+    - gromacs target=neoverse_v1
+    - gromacs target=neoverse_n1
     # - lammps
     # - mpas-model
-    - openfoam
-    - palace
+    # - openfoam
+    # Spack cannot read cflags="", re-enable when https://github.com/spack/spack/issues/39671 is fixed
```

Comment:
See previous comment. It is also unclear which specs this is applying to since none of these have `cflags` specified in the spec string.

Reply:
Done.